ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+# Payment Processor Comparison
+
+Most new merchants will likely only consider the price of the service. Since BTCPay is free, that may have led you here and if so, welcome. 
+
+We want users to understand not only BTCPay, but also how payments can be processed on behalf of a merchant. In order to help users understand trade-offs when using various cryptocurrency payment processing methods. We don't know which payment processors provide which services. That will need to be part of your own research. The following list is a good place to start:
+
+* [Features](#Features)
+* [Security](#Security)
+* [Privacy](#Privacy)
+* [Decentralized](#Decentralized)
+* [Fiat](#Fiat)
+
+----
+
+## Features
+
+Every payment processor has features, here are some BTCPay features:
+
```

Comment:
You forgot the biggest feature which will interest any merchant - no fees. I know you mentioned in the first sentence of the document, but we should clarify why is it free, what sort of fees are not charged.